### PR TITLE
Framework: Update package-lock.json to expected

### LINKED ISSUE #9407: outdated SHA of new package-json-lint breaks npm install
**Describe the bug**
In https://github.com/WordPress/gutenberg/commit/f688989477bb053e763697603243a30c1ad086d4 ; 
npm-package-json-lint was updated from 3.0.0 to 3.3.1 but [its SHA was also not changed in package-lock.json](https://github.com/WordPress/gutenberg/commit/f688989477bb053e763697603243a30c1ad086d4#diff-32607347f8126e6534ebc7ebaec4853dL14419)

I'm not very strong in node but when updating other packages in the past; the sha was changed in package-lock.json (see (https://github.com/WordPress/gutenberg/commit/f24661911fb2829fcf273f7fffa842cdc80dc0ca#diff-32607347f8126e6534ebc7ebaec4853d as example)

As a result, when I run ``npm install`` I receive the error: 

``Verification failed while extracting npm-package-json-lint@3.3.1:
npm ERR! Verification failed while extracting npm-package-json-lint@3.3.1:
npm ERR! sha512-Npd7o3qMkNWcBr/2sjNWJPY7N8911+EIdlqwLpfMOMUNGNo230eJTWA0zVc/PzIlviyO4d5o545cc4hLpIn7jg== integrity checksum failed when using sha512: wanted sha512-Npd7o3qMkNWcBr/2sjNWJPY7N8911+EIdlqwLpfMOMUNGNo230eJTWA0zVc/PzIlviyO4d5o545cc4hLpIn7jg== but got sha512-Wz5byxFKbitRcQS66wCTqrje/uCLJPH+jsEvIV4H2G//E4iB/X0utVPLZzbs4hukoeKuLFadHbMcILa6Cr027w==. (25420 bytes)```

**To Reproduce**
1. clone or switch to master branch
2. run ``npm install``

### PR DESCRIPTION
**Pull request by aduth**
Fixes #9407
Issues introduced in #9321, #8911

This pull request seeks to resolve divergences in installed packages which result in local change or failed `npm install`.

Unsure if there's an easier process, but I undertook the following (very forcefully destructive steps) to remedy:

```
rm -rf node_modules
git clean -fdx
npm cache clean --force
npm install --no-save npm-package-json-lint@3.3.1
npm install
```

**Testing instructions:**

Verify there are no local changes after running `npm install` with a fresh clone with fresh npm cache.

cc @skorasaurus

### CODE REVIEW

**Comment by skorasaurus** [comment]
I ran: 

git checkout --track upstream/fix/package-lock-json-sha
npm install 

that worked as expected! 

but there was a diff as a result - https://gist.github.com/skorasaurus/22d92da910c42c6137caf1baa27b45b0

File: package-lock.json [hunk 1]
```diff
@@ -2010,7 +2002,7 @@
 		"@wordpress/block-library": {
 			"version": "file:packages/block-library",
 			"requires": {
-				"@babel/runtime": "^7.0.0-beta.52",
+				"@babel/runtime-corejs2": "7.0.0-beta.56",
```
Comment (ntwb) [inline package-lock.json]: I'm going to revert this change and restore lines 707-714 above.

Let's have this PR only updatesha512 hash for ``

All the Babel deps have/are being updated for the 7.0.0 release in #9171

**Review by ntwb — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (aduth) [reply package-lock.json]: The change was one which should have been made to a previous dependency update and is present in master (#8911). I think it should have remained.
